lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -1642,6 +1642,40 @@ def test_team_events(self, view_name, event_name, view_data, view_kwargs, mock_r
         self.assertEqual(name, event_name)
         self.assertEqual(event['team_id'], team.team_id)
 
+    @ddt.data(
+        ('vote_for_thread', 'thread_id', 'thread'),
+        ('undo_vote_for_thread', 'thread_id', 'thread'),
+        ('vote_for_comment', 'comment_id', 'response'),
+        ('undo_vote_for_comment', 'comment_id', 'response'),
+    )
+    @ddt.unpack
+    @patch('eventtracking.tracker.emit')
```

Comment:
I understand that you're just updating an older test file to test your particular function, but just for future reference, there is a [useful event testing mixin](https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/testing.py#L62) for any future events testing you might end up doing.